frontend/src/components/SearchResultsItem.js

```diff
@@ -283,74 +253,98 @@ const SearchResultsItem = ({
           // Only show eval data when user is signed in
           style={{
             color: course.average_workload
-              ? workloadColormap(course.average_workload).darken(2).css()
+              ? workloadColormap(course.average_workload).darken(3).css()
               : '#b5b5b5',
             backgroundColor: course.average_workload
-              ? chroma(workloadColormap(course.average_workload))
+              ? workloadColormap(course.average_workload)
               : '#ebebeb',
           }}
           className={Styles.rating_cell + ' m-auto'}
         >
           {course.average_workload ? course.average_workload.toFixed(1) : 'N/A'}
```

Comment:
If all three are N/A, the data looks a bit packed - can we make the N/A text non-bolded and/or a smaller font size

![image](https://user-images.githubusercontent.com/12566801/99138456-98c68500-25fe-11eb-812e-4ff6e00e1baa.png)